src/webpack/webpack.zips.js

```diff
@@ -0,0 +1,16 @@
+const path = require("path")
+const { merge } = require("webpack-merge")
+require("dotenv").config()
+
+const common = require("./webpack.prod.js")
+
+const envDistPath = process.env["COURSE_ZIPS_DIST_PATH"]
+const distPath = path.isAbsolute(envDistPath) ?
+  envDistPath :
+  path.join(__dirname, "..", "..", envDistPath)
+
+module.exports = merge(common, {
+  output: {
+    path: distPath
+  }
+})
```

Comment:
We don't have to make a decision right now, but I think it's worth discussing removing course zip functionality from this repo as it would have to be re-worked anyway based on the new structure.

Reply:
ah I removed some of it (the commands in various places and some directories) meant to remove the rest. I'll just delete this file and then I think we can add back either here or wherever makes sense when we're ready to rebuild that feature.